lib/dbservice_web/controllers/session_controller.ex

```diff
@@ -42,18 +42,40 @@ defmodule DbserviceWeb.SessionController do
   end
 
   def index(conn, params) do
+    sort_order =
+      case params["sort_order"] do
+        "asc" -> :asc
+        # default
+        _ -> :desc
+      end
+
     query =
       from m in Session,
-        order_by: [asc: m.id],
+        order_by: [{^sort_order, m.id}],
         offset: ^params["offset"],
         limit: ^params["limit"]
 
     query =
       Enum.reduce(params, query, fn {key, value}, acc ->
-        case String.to_existing_atom(key) do
-          :offset -> acc
-          :limit -> acc
-          atom -> from u in acc, where: field(u, ^atom) == ^value
+        case key do
+          "offset" ->
+            acc
+
+          "limit" ->
+            acc
+
+          "sort_order" ->
+            acc
+
+          "session_id_is_null" ->
+            case value do
+              "true" -> from u in acc, where: is_nil(u.session_id)
+              "false" -> from u in acc, where: not is_nil(u.session_id)
```

Comment:
What's the use case of this condition?

Reply:
i need two tables on frontend... one where session id is null .. and one where it is not null